src/Microsoft.Framework.Runtime.Roslyn/RoslynDiagnostics.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Framework.Runtime.Roslyn
+{
+    internal class RoslynDiagnostics
+    {
+        internal static readonly DiagnosticDescriptor StrongNamingNotSupported = new DiagnosticDescriptor(
```

Comment:
The are indeed still `internal abstract` in Roslyn. I'm leave this as-is until we get an implementable strong name system. Not using a `StrongNameProvider` yields "CS7027: Error signing ... Assembly signing not supported."